### ISSUE TITLE
Ensure cross-browser support or fallbacks for flexbox

### ISSUE THREAD

**Issue by jeremyboggs** [scholarslab/takeback#1]
We use flexbox on the theme header, but we should test that this works, or degrades gracefully, in browsers that do not support it.

There's also a polyfill that could be employed—[Flexie](http://flexiejs.com/)

**Comment by jeremyboggs** [comment]
Removes reference to Flexie polyfill. That particular project is no longer being maintained, and newer-ish development incorporating the updated Flexbox spec is on [Reflexie](https://github.com/doctyper/reflexie).

**Comment by jeremyboggs** [comment]
@emily-em Let's test the site on the current version and the previous version of every modern browser, for both Windows and Mac, and confirm Flexbox is supported and the site layout works. If so, we'll close this ticket.

**Comment by jeremyboggs** [comment]
- [x]  Firefox
- [x] Chrome
- [ ] Safari
- [x] Opera
- [ ] Edge
- [ ] Internet Explorer

**Comment by emily-em** [comment]
Chrome, Firefox, and Opera work fine for me, though Opera has a popup warning saying that we have content from insecure sources. Safari gives me a 404 not found message. Will try Edge and Internet Explorer with a Windows computer when I get a chance.

**Comment by emily-em** [comment]
I tested several pages (home, about, browse items, and look at item) on the take back site on various browsers on my computer (macbook) and a library computer (old windows) and got the following results:
- Safari: Will not open using my safari (gets 404 not found message)
- Chrome: works on both computers
- Firefox: works on both computers
- Opera: works on my computer but says we have content from insecure sources
- IE: works more or less, but looks weird. Color is not set up right, there a white bars on the left side and spacing is off.
- Edge: not able to use on computers available to me.

**Comment by jeremyboggs** [comment]
> IE: works more or less, but looks weird. Color is not set up right, there a white bars on the left side and spacing is off.

You know which version of IE?

**Comment by jeremyboggs** [comment]
> Safari: Will not open using my safari (gets 404 not found message)

This seems like a caching issue in your browser. try clearing it entirely. If you got 404s in multiple browsers, it would indicate a problem on the server.

**Comment by emily-em** [comment]
Yes! You were right. I just cleared cache and Safari works fine now. The Internet Explorer version I was using was 11.0.9600 (maybe more or fewer zeros, but that was the gist)
